backend/config/default.cjs

```diff
@@ -241,4 +242,8 @@ module.exports = {
       },
     },
   },
+
+  inference: {
+    tokenHeader: undefined,
```

Comment:
If you don't want to give an example value, I personally would prefer to put an empty string here as it is clearer what data type should be used.